ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -21,18 +48,54 @@ def add_arguments(self, parser):
         parser.add_argument('--dry-run', action='store_true',
                             help='Do not really create the package.')
 
+        group = parser.add_argument_group('yaml generation arguments')
+        group.add_argument('--package-id', default="<package_id>",
+                           help="Identifier of the package.")
+        group.add_argument('--name', default="<Package name>",
+                           help="Name of the package.")
+        group.add_argument('--description', default="<Package description>",
+                           help="Description of the package.")
+        group.add_argument('--language', default="<lang>",
+                           help="Language of the package.")
+        group.add_argument('--url', default="<http://...>",
+                           help="URL where the package we will be available")
```

Comment:
I think @letompouce meant you had mixed `we` and `will`.

That is, you should have replaced `we` by `will`, not added `will`. 😄